package.json

```diff
@@ -27,7 +27,7 @@
     "react-with-direction": "1.3.0",
     "replace-x": "1.5.0",
     "singularitygs": "^1.7.0",
-    "solr-faceted-search-react": "https://github.com/palantirnet/solr-faceted-search-react"
+    "solr-faceted-search-react": "git+ssh://git@github.com:palantirnet/solr-faceted-search-react#root-138-autocomplete"
```

Comment:
This will need to change before merge.

Reply:
I'll update when you're ready.

Reply:
Should we also [merge the dependency pr](https://github.com/palantirnet/solr-faceted-search-react/pull/7) and then the commit hash from `yarn.lock` in this project should point to that merge commit?

I'll be creating a follow up branch in both repositories based on the solr config work from yesterday, but I think it'd be good to have working snapshots from merges.

Reply:
Yes, I believe so. One thing I'm not clear on is how this affects public releases of the project. (Right now, the Drupal module pulls a build from CDN.) We shouldn't release a new version to CDN yet, I don't think, and when we do, we'll need to publish a change log.

Reply:
@agentrickard I've updated the dependency back to master after merging https://github.com/palantirnet/solr-faceted-search-react/pull/7

The next step is to publish a release of this repo.  I'll draft one and send you a link prior to publishing.